defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
-nodejs_playbook_version: "0.1.5"
+nodejs_playbook_version: "0.1.6"
+nodejs_install_source: build #One of: build (from source), nodesource_debs (works only on debian-based distros)
```

Comment:
I think we should change the var to `nodejs_install_method: source # options: {source|binary}`. So for binary it would be `nodejs_install_method: binary`.